sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -53,12 +53,12 @@ az cognitiveservices account create \
 ### Include the Package
 **Note:** This version targets Azure Text Analytics service API version v3.0.
 
-[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;dependency})
+[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-ai-textanalytics</artifactId>
-    <version>5.0.4</version>
+    <version>5.1.0-beta.6</version>
```

Comment:
this should point to latest stable release avsilable.

Reply:
I think last time Vinay we decided to go with the releasing version.
As when publishing these docs it would be odd to have the latest beta readme (code snippets) but the version would not be compatible with the rest of the readme (patch- latest)

Reply:
As an end user, I would like to use a stable release and it would be odd to see a beta release being recommended to me.
Plus, Customers don't pay for any support on the beta releases.
The README samples should only be updated when the beta changes are about to GA.

Reply:
Readme updates provide a chance to showcase the beta features and get feedback on the same, and getting that feedback closer to GA wouldn't help much as we would be down many beta's by then. I will check with other langs on their approach and can follow the same.